cpp/CMake/FindLMDB.cmake

```diff
@@ -0,0 +1,33 @@
+# Taken from phabricator (Apache License 2.0)
```

Comment:
I think this should be added to our [NOTICE.txt](https://github.com/man-group/ArcticDB/blob/master/NOTICE.txt) file.

Reply:
It seems that this file originates from [the sink project](https://invent.kde.org/pim/sink/-/blob/master/cmake/modules/FindLMDB.cmake) which has no license.

Since this file brings a relatively generic configuration, we can omit the mention while keeping the copyrights at the top?